abipy/abio/tests/test_factories.py

```diff
@@ -92,6 +90,61 @@ def test_g0w0_with_ppmodel_inputs(self):
         flow.allocate()
         #flow.make_scheduler().start()
 
+    def test_convergence_inputs_single(self):
+        """Testing g0w0_convergence_input factory single calculation."""
+        scf_kppa, scf_nband, nscf_nband = 10, 10, [10]
+        ecuteps, ecutsigx = [2], 2
+
+        inputs = g0w0_convergence_inputs(self.si_structure, self.si_pseudo, scf_kppa, nscf_nband, ecuteps, ecutsigx,
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Unexpected keyword argument 'ecut_s' in function call](https://www.codacy.com/app/gmatteo/abipy/file/2694273838/issues/source?bid=3176346&fileBranchId=3340437#l98)